docs/api/app.md

```diff
@@ -343,6 +343,10 @@ app.on('login', (event, webContents, request, authInfo, callback) => {
 })
 ```
 
+### Event: 'gpu-info-update'
```

Comment:
Suggestion: add some brief words about why this event may be needed.

Reply:
I've added a note to `app.getGPUFeatureStatus()`